### ISSUE TITLE
[Bug] Several firmware packages use signature from outdated (?) keys

### ISSUE THREAD

**Issue by BrainwreckedTech** [chaotic-aur/packages#2137]
### What happens?

When trying to install aic94xx-firmware, upd72020x-fw, or wd719x-firmware from chaotic-aur, pacman complains that the signatures are invalid

### What is expected to happen?

Pacman allows installation after verifying signatures are valid.

### If possible, please attach logs

```
(567/567) checking keys in keyring                                                                  [----------------------------------------------------------] 100%
(567/567) checking package integrity                                                                [----------------------------------------------------------] 100%
error: aic94xx-firmware: signature from "Pedro Henrique Lara Campos <root@pedrohlc.com>" is invalid
:: File /var/cache/pacman/pkg/aic94xx-firmware-30-9-any.pkg.tar.zst is corrupted (invalid or corrupted package (PGP signature)).
Do you want to delete it? [Y/n] 
error: upd72020x-fw: signature from "Pedro Henrique Lara Campos <root@pedrohlc.com>" is invalid
:: File /var/cache/pacman/pkg/upd72020x-fw-20200826-3-any.pkg.tar.zst is corrupted (invalid or corrupted package (PGP signature)).
Do you want to delete it? [Y/n] 
error: wd719x-firmware: signature from "Nico Jensch (Chaotic-AUR) <dr460nf1r3@garudalinux.org>" is invalid
:: File /var/cache/pacman/pkg/wd719x-firmware-1-7-any.pkg.tar.zst is corrupted (invalid or corrupted package (PGP signature)).
Do you want to delete it? [Y/n] 
error: failed to commit transaction (invalid or corrupted package)
Errors occurred, no packages were upgraded.
==> ERROR: Failed to install packages to new root
ls: cannot access '/tmp/archiso-out/': No such file or directory
mv: cannot stat '/tmp/archiso-out/*.iso': No such file or directory
+ rm -r /tmp/archiso-work /tmp/archiso-out
rm: cannot remove '/tmp/archiso-out': No such file or directory
```

**Comment by Technetium1** [comment]
@BrainwreckedTech I was able to download them all successfully with `pacman -Sw aic94xx-firmware upd72020x-fw wd719x-firmware`. Is this still a problem?

**Comment by BrainwreckedTech** [comment]
I just tested this again, and creating the custom Arch ISO worked this time.

```
Total Download Size:     19.61 MiB
Total Installed Size:  3014.49 MiB

:: Proceed with installation? [Y/n] 
:: Retrieving packages...
 systemd-252.2-3-x86_64                                                   7.4 MiB  2.92 MiB/s 00:03 [----------------------------------------------------------] 100%
 glib2-2.74.3-1-x86_64                                                    3.7 MiB  1126 KiB/s 00:03 [----------------------------------------------------------] 100%
 graphviz-7.0.4-1-x86_64                                                  3.6 MiB   923 KiB/s 00:04 [----------------------------------------------------------] 100%
 oh-my-zsh-git-r6724.7ea8a93bb-1-any                                   1000.7 KiB  1334 KiB/s 00:01 [----------------------------------------------------------] 100%
 sqlite-3.40.0-2-x86_64                                                1606.3 KiB   385 KiB/s 00:04 [----------------------------------------------------------] 100%
 broadcom-wl-6.30.223.271-433-x86_64                                   1473.9 KiB   343 KiB/s 00:04 [----------------------------------------------------------] 100%
 systemd-libs-252.2-3-x86_64                                            641.5 KiB  1734 KiB/s 00:00 [----------------------------------------------------------] 100%
 usbutils-015-2-x86_64                                                   88.2 KiB   304 KiB/s 00:00 [----------------------------------------------------------] 100%
 libice-1.1.0-1-x86_64                                                   78.4 KiB   523 KiB/s 00:00 [----------------------------------------------------------] 100%
 xorg-xprop-1.2.6-1-x86_64                                               25.4 KiB   509 KiB/s 00:00 [----------------------------------------------------------] 100%
 xorg-xset-1.2.5-1-x86_64                                                19.1 KiB   381 KiB/s 00:00 [----------------------------------------------------------] 100%
 aic94xx-firmware-30-9-any                                               24.8 KiB   248 KiB/s 00:00 [----------------------------------------------------------] 100%
 pacman-mirrorlist-20221204-1-any                                         7.8 KiB  78.0 KiB/s 00:00 [----------------------------------------------------------] 100%
 wd719x-firmware-1-7-any                                                 10.5 KiB  70.2 KiB/s 00:00 [----------------------------------------------------------] 100%
 systemd-sysvcompat-252.2-3-x86_64                                        6.0 KiB   100 KiB/s 00:00 [----------------------------------------------------------] 100%
 systemd-resolvconf-252.2-3-x86_64                                        3.1 KiB  52.3 KiB/s 00:00 [----------------------------------------------------------] 100%
 upd72020x-fw-20200826-3-any                                             19.8 KiB  66.1 KiB/s 00:00 [----------------------------------------------------------] 100%
 Total (17/17)                                                           19.6 MiB  4.11 MiB/s 00:05 [----------------------------------------------------------] 100%
(567/567) checking keys in keyring                                                                  [----------------------------------------------------------] 100%
(567/567) checking package integrity                                                                [----------------------------------------------------------] 100%
(567/567) loading package files                                                                     [----------------------------------------------------------] 100%
(567/567) checking for file conflicts                                                               [----------------------------------------------------------] 100%
:: Processing package changes...
(  1/567) installing aic94xx-firmware                                                               [----------------------------------------------------------] 100%
...
[mkarchiso] INFO: Done!
1.5G    /tmp/archiso-out/cai-full-2022.12.04-x86_64.iso
total 1.5G
-rw-r--r-- 1 root root 1.5G Dec  4 21:47 cai-full-2022.12.04-x86_64.iso
copied '/tmp/archiso-out/cai-full-2022.12.04-x86_64.iso' -> '/srv/files/os/iso/cai-full-2022.12.04-x86_64.iso'
removed '/tmp/archiso-out/cai-full-2022.12.04-x86_64.iso'
+ rm -r /tmp/archiso-work /tmp/archiso-out
```

This wasn't an issue back on November 2nd when I made the last ISO.  But it came up November 20 when I used that custom ISO to install Arch in a VM.  Figured it was just a key switcheroo thing with the live media that would sort itself out come December when I'd just roll a new custom ISO.  On December 2nd, I couldn't roll the "full" version because those had extra firmware from the chaotic-aur repo.

TL;DR: Everything works now.

**Comment by Technetium1** [comment]
Does this always report the same result? It might have been a temporary issue with a mirror you were redirected to. https://status.chaotic.cx/geo

**Comment by BrainwreckedTech** [comment]
For me, https://status.chaotic.cx/geo Seems to bounce around all of the VA mirrors.

**Comment by Technetium1** [comment]
@BrainwreckedTech if you manually choose a mirror instead of geo-mirror, it should make it easier to identify what caused this in the future.

**Comment by BrainwreckedTech** [comment]
I just did a `pacman -Si` on the affected packages and have noticed the following changes:

`upd72020x-fw` and `aic94xx-firmware` changed packager
from Pedro Henrique Lara Campos <root@pedrohlc.com>
to UFSCar HPC Builder <hpc.ufscar@chaotic.cx>

`wd719x-firmware` changed packager
from Nico Jensch (Chaotic-AUR) <dr460nf1r3@garudalinux.org>
to Nico Jensch <dr460nf1r3@chaotic.cx>

That last one leads me to believe there was some oversight WRT package signing keys, where `chaotic-keyring` distrusted the old maintainer keys (or they just expired on their own) but the packages weren't re-signed with the new maintainer key.  So I don't think this was a mirror-specific problem.
